jumanji/training/configs/env/tsp.yaml

```diff
@@ -24,5 +24,5 @@ a2c:
     bootstrapping_factor: 0.95
     l_pg: 1.0
     l_td: 1.0
-    l_en: 0.01
+    l_en: 0.005
```

Comment:
Btw you are giving some numbers like this as decimals and others in scientific notation, any reason?

Reply:
No reason, I don't have a strong opinion at all. Please feel free to suggest changes where you think it should be different. Otherwise, I will go with what is in if you don't mind.